pkg/api/registry/registry.proto

```diff
@@ -22,61 +22,41 @@ message Destination {
     uint32 weight = 2;
 }
 
-message NetworkServiceManager {
-    string name = 1;
-    string url = 2;
-    google.protobuf.Timestamp expiration_time = 3;
-    string state = 4;
-}
-
 message NetworkServiceEndpoint {
     string name = 1;
     string payload = 2;
```

Comment:
Do we need payload here?

Reply:
Yes, you are right it is not needed, will remove